build/bin/entrypoint

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+# This is documented here:
+# https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html#openshift-specific-guidelines
+
+if ! whoami &>/dev/null; then
+  if [ -w /etc/passwd ]; then
+    echo "kubedirector:x:$(id -u):$(id -g):kubedirector user:/home/kubedirector:/sbin/nologin" >> /etc/passwd
```

Comment:
You can also use $USER_HOME. I see that is being set in Dockerfile
Similarly for $USER_NAME also

Reply:
Ah true. I think that's what the operator-sdk generated examples do. I forgot that those ENV values would be available here too.